README.md

```diff
@@ -19,35 +19,38 @@
 
 </div>
 
-# Here is our website to collect data:
-
-[open-assistant.io](https://open-assistant.io)
-
-(project documentation lives [here](https://laion-ai.github.io/Open-Assistant/))
-
 # Table of Contents
 
 - [What is Open Assistant?](#what-is-open-assistant)
-- [Do you want to try it out?](#do-you-want-to-try-it-out)
-- [The Plan](#the-plan)
+- [Useful Links](#useful-links)
+- [How To Try It Out?](#how-to-try-it-out)
 - [The Vision](#the-vision)
-- [How can you help?](#how-can-you-help)
-- [I’m in! How do I contribute?](CONTRIBUTING.md)
+- [The Plan](#the-plan)
+- [How You Can Help](#how-you-can-help)
 
 ---
 
 ## What is Open Assistant?
 
-<p align="center">
-    Open Assistant is a project meant to give everyone access to a great chat based large language model.
-</p>
+Open Assistant is a project meant to give everyone access to a great chat based
+large language model.
 
 We believe that by doing this we will create a revolution in innovation in
 language. In the same way that stable-diffusion helped the world make art and
 images in new ways we hope Open Assistant can help improve the world by
 improving language itself.
 
-## Do you want to try it out?
+# Useful Links
+
+- Project Repository -
```

Comment:
We could just make the text itself hyperlinks for these. I also don't see any reason to link to the project repository from the README in the root of the project repository so we could remove that one

Reply:
I'm guessing you mean something like this? `[Project Repository](whateverlinkitis)`?

Reply:
But remove the project repo link 😅